clients/telemetry/src/FrontendTelemetryClient.ts

```diff
@@ -25,12 +25,6 @@ export class ClientTelemetryEvent extends TelemetryEvent {
   public readonly clientApplicationId?: string;
   /** Application version configured for the client application. */
   public readonly clientApplicationVersion?: string;
-  /** Client-configured ID of the current user */
-  public readonly clientUserId?: string;
-  /** Client-configured ID of the current user's organization */
-  public readonly clientUserOrgId?: string;
-  /** Client-configured name of the current user's organization */
-  public readonly clientUserOrgName?: string;

```

Comment:
I'd like to avoid assuming that we can log these in telemetry events. There are privacy concerns about when and where these should be logged at in plain text. I'd like to follow up on telemetry client going forward for 3.0 anyway so maybe lets tackle that in a separate PR?